### ISSUE TITLE
'Report which rules apply' function, or 'trace rule' function, + total words changed

### ISSUE THREAD

**Issue by MrSquiffy** [def-gthill/lexurgy#32]
Being able to quickly see at a glance where & how many times rules apply to words is incredibly useful, especially when you're applying this to an entire lexicon of 100s+ words, and for quickly identifying abnormalities generated by a given rule. 

It's about the only feature that Zompist SCA has over this program at this point - if this was added, his SCA would be rendered completely obsolete.

Is this something that could be considered looking into for the future?

**Comment by def-gthill** [comment]
I agree that this would be really useful! This is the kind of thing I was going for with the "trace word" and "stop before" features, but I can see how neither quite gets you quick look at what one rule is doing.

Thanks for the suggestion!
